docs/book/src/reference/reference.md

```diff
@@ -32,5 +32,8 @@
   - [Artifacts](artifacts.md)
   - [Writing controller tests](writing-tests.md)
   - [Metrics](metrics.md)
+
+      - [Metrics Reference](metrics-reference.md)
```

Comment:
```suggestion
      - [Reference](metrics-reference.md)
```
It is inside of the Metrics so we do not need to use twice the name.

Reply:
Gotcha 👍🏽